src/styles/blocks/_container.scss

```
@@ -0,0 +1,27 @@
+.container {
+display: grid;
+margin: 0 auto;
+box-sizing: border-box;
+justify-content: center;
+gap: 46px 48px;
+padding: 50px 40px;
+grid-template-columns: 200px;
+
+
+@media (min-width: 488px) {
+grid-template-columns: repeat(2, 200px);
+}
+
+
+@media (min-width: 768px) {
+grid-template-columns: repeat(3, 200px);
+}
+
+
+@media (min-width: 1024px) {
+grid-template-columns: repeat(4, 200px);
+}
```

please fix formatting

```suggestion
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, 200px);
  }
  
  
  @media (min-width: 768px) {
    grid-template-columns: repeat(3, 200px);
  }
  
  
  @media (min-width: 1024px) {
    grid-template-columns: repeat(4, 200px);
  }
```